tools/istio-iptables/pkg/utils/utils.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Istio Authors
```

Comment:
I wouldn't create a whole new package just for a 7-line function.
Move this file into `.../pkg/cmd/format.go`?

Reply:
done